tests/components/bayesian/test_binary_sensor.py

```diff
@@ -322,7 +323,9 @@ def test_multiple_observations(self):
 
         state = self.hass.states.get("binary_sensor.test_binary")
 
-        assert [None, None] == state.attributes.get("observations")
+        for key, attrs in state.attributes.items():
+            json.dumps(attrs)
+        assert [] == state.attributes.get("observations")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5403678)